packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -16,12 +18,14 @@ export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
           ...nextObject,
         }
 
-        if (key === 'custom') {
-          return {
-            ...merged,
-          }
+        if (key === CUSTOM) {
+          return merged
         }
 
+        /**
+         * Check if `oldObj` value is an array and that same array exist on `newObj` with more than one element
+         * then Loop over `newObj` array to copy empty array props from `oldObj` to `newObj`
```

Comment:
maybe discuss a real life use case to describe what this solves would help describe part of the feature as well.

Reply:
so you mean, it would be better to explain the reason of the code, something like `Loop through the elements in newObj and make sure to copy undefined array props from oldObj to ignore errors when comparing arrays`